src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import random
+from typing import List, MutableMapping, Set
```

Comment:
If you add `from __future__ import annotations` here you can type things with the builtin `set` and `list` instead of `Set` and `List` respectively.